contracts/periphery/Periphery.sol

```diff
@@ -176,10 +176,16 @@ contract Periphery is IPeriphery {
 
         // if WETH pools, accept deposit only in ETH
         if (address(underlyingToken) == address(_weth)) {
-            require(marginDelta <= 0, "INV");
+            require(
+                marginDelta <= 0,
+                "Invalid: cannot deposit wETH to this pool"
```

Comment:
long reason strings are expensive in Solidity. They should be kept as short as possible. 
https://github.com/ethereum/solidity/issues/4588